### ISSUE TITLE
Submit apple - rejected

### ISSUE THREAD

**Issue by rodrigors3012** [Simbul/baker#462]
after submitting a project with the bank got the following response:
or know someone knows how to solve the bank will support it?

```
2.23

We found that your app does not follow the iOS Data Storage Guidelines, which is required per the App Store Review Guidelines.

To check how much data your app is storing:

- Install and launch your app
- Go to Settings > iCloud > Storage & Backup > Manage Storage
- If necessary, tap "Show all apps"
- Check your app's storage

The iOS Data Storage Guidelines indicate that only content that the user creates using your app, e.g., documents, new files, edits, etc., should be backed up by iCloud.

Temporary files used by your app should only be stored in the /tmp directory; please remember to delete the files stored in this location when the user exits the app.

Data that can be recreated but must persist for proper functioning of your app - or because customers expect it to be available for offline use - should be marked with the "do not back up" attribute. For NSURL objects, add the NSURLIsExcludedFromBackupKey attribute to prevent the corresponding file from being backed up. For CFURLRef objects, use the corresponding kCFURLIsExcludedFromBackupKey attribute.

For more information, please see Technical Q&A 1719: How do I prevent files from being backed up to iCloud and iTunes?.

It is necessary to revise your app to meet the requirements of the iOS Data Storage Guidelines.
For discrete code-level questions, you may wish to consult with Apple Developer Technical Support. Please be sure to:

- include the complete details of your rejection issues
- prepare any symbolicated crash logs, screenshots, and steps to reproduce the issues for when the DTS engineer follows up.

For information on how to symbolicate and read a crash log, please see Tech Note TN2151 Understanding and Analyzing iPhone OS Application Crash Reports.

If you have difficulty reproducing this issue, please try testing the workflow as described in <https://developer.apple.com/library/ios/qa/qa1764/>Testing Workflow with Xcode's Archive feature".
```

**Comment by folletto** [comment]
What do you mean by "the bank"?
Could you please tell us what your app does, what you changed, what version of Baker are you using? It's quite impossible to help you without informations, please give us more details so we can try to see if there's a solution. :)

Thanks!

**Comment by rodrigors3012** [comment]
this project: https://github.com/nin9creative/baker
Version: 3.0

**Comment by folletto** [comment]
I'm afraid then we can't help much, you should ask them since it's a fork of Baker.

We'll still check and see if that issue is present on our side as well, but unfortunately the two projects are separate right now.

**Comment by folletto** [comment]
We have fixed it on our branch with this commit: ca5ec939d1cec47f4edbe3ff15a9d1a0347a3744
You could probably refer this to them so they can fix it too. :)
